code/datums/diseases/transformation.dm

```diff
@@ -127,9 +127,11 @@
 	if(affected_mob.mind && !is_monkey(affected_mob.mind))
 		add_monkey(affected_mob.mind)
 	if(ishuman(affected_mob))
-		if(affected_mob && !is_monkey_leader(affected_mob.mind))
-			var/mob/living/carbon/monkey/M = affected_mob.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPVIRUS | TR_KEEPSE)
+		GLOB.huds[DATA_HUD_MEDICAL_ADVANCED].add_hud_to(affected_mob)
```

Comment:
This hud isn't needed anymore because monkeys are given a hud in their code part and gorillas are given their hud in their code part too

Reply:
Correct. Fixed.